### PR TITLE
[infra] eloquent upgrade

### PR DESCRIPTION
**Pull request by stonier**
* Upgrades for latest ament
* Explicitly call out `std_msgs/Header` instead of `Header`
* Ament forces only uppercase or lowercase on variable names
* Drop redundant `builtin_interfaces`

### CODE REVIEW

File: CMakeLists.txt [hunk 1]
```diff
@@ -1,32 +1,70 @@
...
+
+if(CMAKE_COMPILER_IS_GNUCXX OR CMAKE_CXX_COMPILER_ID MATCHES "Clang")
+  add_compile_options(-Wall -Wextra -Wpedantic)
+endif()
+
+##############################################################################
+# Dependencies
+##############################################################################
+
+find_package(ament_cmake REQUIRED)
```
Comment (clalancette) [inline CMakeLists.txt:22]: Nit: alphabetize.

File: msg/KeyboardInput.msg [hunk 2]
```diff
@@ -2,12 +2,12 @@
...
+uint8  KEYCODE_LEFT     = 68     # 0x44
+uint8  KEYCODE_UP       = 65     # 0x41
+uint8  KEYCODE_DOWN     = 66     # 0x42
+uint8  KEYCODE_SPACE    = 32     # 0x20
+uint8  KEYCODE_ENABLE   = 101    # 0x65, 'e'
+uint8  KEYCODE_DISABLE  = 100    # 0x64, 'd'
 
-uint8 pressedKey
\ No newline at end of file
+uint8 pressed_key
```
Comment (clalancette) [inline msg/KeyboardInput.msg]: Nit: add newline at end of file.

File: msg/ScanAngle.msg [hunk 3]
```diff
@@ -1,2 +1,2 @@
-Header header
+std_msgs/Header header
 float64 scan_angle
```
Comment (clalancette) [inline msg/ScanAngle.msg]: Ditto.

**Review by clalancette — APPROVED** [review]
A couple of nits, but I'll approve.

File: msg/ScanAngle.msg (hunk 3 above)
Comment (stonier) [reply msg/ScanAngle.msg]: Fixed in ac1e25e9

File: msg/KeyboardInput.msg (hunk 2 above)
Comment (stonier) [reply msg/KeyboardInput.msg]: Fixed in ac1e25e9

File: CMakeLists.txt (hunk 1 above)
Comment (stonier) [reply CMakeLists.txt:22]: Fixed in ac1e25e9

**Pull request merged**